toolkit/tools/specreader/specreader.go

```diff
@@ -63,13 +66,23 @@ func main() {
 		logger.Log.Panicf("Value in --workers must be greater than zero. Found %d", *workers)
 	}
 
-	err := parseSPECsWrapper(*buildDir, *specsDir, *rpmsDir, *srpmsDir, *distTag, *output, *workerTar, *workers, *runCheck)
+	var toolchainRPMs []string
+	var err error
+	toolchainManifestPath := *toolchainManifest
+	if len(toolchainManifestPath) > 0 {
```

Comment:
Is the check necessary, since we're using `ExistingFile()` on the `toolchain-manifest` argument?